docs/book/src/topics/configuration.md

```diff
@@ -0,0 +1,26 @@
+# Configuration
+
+This page describes how to configure CAPZ installations.
+
+## Environment Variables
+
+These environment variables are used to template the kustomize output for CAPZ control plane installations.
```

Comment:
Would be good to let people know they must set these environment variables before `clusterctl init` OR if they need to modify/add them afterwards how to do so.